website/interfaces/src/kafka-payload.ts

```diff
@@ -0,0 +1,5 @@
+export interface KafkaPayload<T> {
+  headers: any;
```

Comment:
if it is http headers, we could describe its type as `Record<string, string>`

Reply:
No, it isn't